PublishedModules/mjlill/LocalButtonPress/requirements.txt

```diff
@@ -0,0 +1 @@
+RPI.GPIO
```

Comment:
This file doesn't belong to modules. Requirements are to be inserted in install file and readme